src/secrets.ts

```diff
@@ -7,6 +7,8 @@ export const BUCKET_URL = process.env.REACT_APP_BUCKET_URL;
 export const STRIPE_PUBLISHABLE_KEY =
   process.env.REACT_APP_STRIPE_PUBLISHABLE_KEY;
 export const BUGSNAG_API_KEY = process.env.REACT_APP_BUGSNAG_API_KEY;
+export const REACT_APP_GA_MEASUREMENT_ID =
+  process.env.REACT_APP_GA_MEASUREMENT_ID ?? "";
```

Comment:
Do you need the fallback value here? The rest of env variables don't use this syntax, keep it consistent unless there is a specific reason for this change.

Reply:
fixed in f519c171fe40a2d4ec6a2642b8f20ce52e3dd58b